### ISSUE TITLE
Home-Assistant forum passkey doesn't support 1Password and will lock users out

### ISSUE THREAD

**Issue by gravelfreeman** [home-assistant/organization#7]
I wanted to bring attention to an issue I've encountered with using passkeys from 1Password with my Home-Assistant forum account.

To provide some context, I'm uncertain about where to raise this issue, as there doesn't seem to be a dedicated repository for the HA forums. Please feel free to redirect this issue to the appropriate repo or guide me to the right resource. I initially attempted to address this concern on Discord, but unfortunately, I wasn't able to make progress.

**Over the past few months, I've been using passkeys from 1Password without any issues. However, I recently decided to deactivate the 2FA code and solely rely on the passkey itself. It seems that the passkeys from 1Password are not functioning correctly with Home-Assistant forum accounts, leading to a situation where users get locked out.**

As a result, I am currently unable to access my account and would greatly appreciate assistance in resolving this critical issue.

**Comment by gravelfreeman** [comment]
This is quite an important security issue locking users out of their accounts. I would really appreciate some help in recovering my account.

Thank you.

**Comment by fabaff** [comment]
On an user account for community.home-assistant.io with the same handle as your GitHub account is Two-Factor Authentication still activated.

**Comment by gravelfreeman** [comment]
> On an user account for community.home-assistant.io with the same handle as your GitHub account is Two-Factor Authentication still activated.

False, I haven't linked my Github account to my community.home-assistant.io account. I tried logging in with github in case and it's offering me to create a new account.

I've setup my passkey on community.home-assistant.io

The error I get is `The security key authentication process failed due to an invalid response.`

**Comment by fabaff** [comment]
I was not talking about liked accounts (or Associated Accounts in Discourse terminology). community.home-assistant.io has a "gravelfreeman" account and that account still has Two-Factor Authentication activated.

Discourse has announced passkeys supports at the end of November 2023. I'm wondering how one was able to use passkeys "Over the past few months...".

**Comment by gravelfreeman** [comment]
> I was not talking about liked accounts (or Associated Accounts in Discourse terminology). community.home-assistant.io has a "gravelfreeman" account and that account still has Two-Factor Authentication activated.
> 
> Discourse has announced passkeys supports at the end of November 2023. I'm wondering how one was able to use passkeys "Over the past few months...".

Ah right, yes my account name on _community.home-assistant.io_ is the same as my Github name.

I'm afraid that this ticket never got a reply from the devs and up to this day, I can't still access my account.

This is problematic.
